cohorts/01_july2018/mike_mitchell/profile.js

```diff
@@ -0,0 +1,53 @@
+let alumni_template = { // 👈 mike_mitchell
+  first_name: 'Mike',
+  last_name: 'Mitchell',
+  email: 'mimitch@mac.com',
+  github: 'https://github.com/mimitch2', // acount url
+  github_username: 'mimitch2',
+  linkedIn: 'https://www.linkedin.com/in/mike-mitchell-706a9117/',
+  portfolio: '', // portfolio url
+  projects: [
+    {
+      name: 'ACA-Job-Tracker'
+      link: 'https://aca-alumni.github.io/ACA-Job-Tracker' // project url
+      github: 'https://github.com/ACA-Alumni/ACA-Job-Tracker' // github repository url
+      summary: '...loads of fun!' // 2-3 sentences max.
+      tools: [
+        'html',
+        'css',
+        '...'
```

Comment:
including '...' will be adding this a tool used in this project. Is that what'cha want?🤔